src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -20498,10 +20498,12 @@ public class C4 : I2, I3
 
             void assertExplicitInterfaceImplementations(INamedTypeSymbol c)
             {
-                var cMabImplementations = ((IMethodSymbol)c.GetMember("I1<(System.Int32a,System.Int32b)>.M")).ExplicitInterfaceImplementations;
+                var members = c.GetMembers("I1<System.ValueTuple<System.Int32,System.Int32>>.M");
```

Comment:
>I1<System.ValueTuple<System.Int32,System.Int32>> [](http://example.com/codeflow?start=44&length=48)

I am not sure if change in the member name is desirable.

Reply:
Originally you proposed to change all explicitly implemented member names, so why is this one a problem?

Reply:
> Originally you proposed to change all explicitly implemented member names, so why is this one a problem?

I do not know the reason behind the change. Hence the concern.